modules/geo-layers/src/mvt-layer/coordinate-transform.js

```diff
@@ -0,0 +1,50 @@
+import {lerp} from 'math.gl';
+
+const availableTransformations = {
+  Point,
+  MultiPoint,
+  LineString,
+  MultiLineString,
+  Polygon,
+  MultiPolygon
+};
+
+function Point([pointX, pointY], bbox, viewport) {
+  const [minX, minY] = viewport.projectFlat([bbox.west, bbox.north]);
+  const [maxX, maxY] = viewport.projectFlat([bbox.east, bbox.south]);
+  const x = lerp(minX, maxX, pointX);
```

Comment:
These are the same values for the same tile. You are repeating the calculation for every point in the geometry.